lms/djangoapps/courseware/tests/test_submitting_problems.py

```diff
@@ -540,14 +554,20 @@ def test_submissions_api_overrides_scores(self):
         self.check_grade_percent(0.67)
         self.assertEqual(self.get_grade_summary()['grade'], 'B')
 
-        # But now we mock out a get_scores call, and watch as it overrides the
-        # score read from StudentModule and our student gets an A instead.
-        with patch('submissions.api.get_scores') as mock_get_scores:
-            mock_get_scores.return_value = {
-                self.problem_location('p3').to_deprecated_string(): (1, 1)
-            }
-            self.check_grade_percent(1.0)
-            self.assertEqual(self.get_grade_summary()['grade'], 'A')
+        # But now, set the score with the submissions API and watch
+        # as it overrides the score read from StudentModule and our
+        # student gets an A instead.
+        self._stop_signal_patch()
```

Comment:
Does this action only affect the scope of this test, or should you restart the patch at the end of this function?